public/js/controllers/game.js

```diff
@@ -277,6 +289,23 @@ angular.module('mean.system') //eslint-disable-line
             }
           }
         }
+
+
+        $scope.sendChatMessage = function (message) {
+          if (message) {
+            $scope.chats.$add({
+              image: game.players[game.playerIndex].avatar,
```

Comment:
Are we still using _avatars_ or the _imageUrl_ for the user that is logged in?

Reply:
No we are not using avatars. When a user signs up, his image url is stored in this location. When he plays the game, his image is uploaded. When a guest plays the game, the avatar url is what is found at this location and used as an image.

Reply:
Okay. Thank you for providing clarity